File: lysh.py
#!/usr/bin/env python

import os
import pandas
import sys

search = ' '.join(sys.argv[1:])
libx = pandas.read_excel(os.path.expanduser('~/Documents/Library.xlsx'))
lib = libx.to_csv(index=False)
lib = lib.split('\n')
for line in lib:
    if search in line:
        print(line)

File: lysh.rb
#!/usr/bin/env ruby

require 'roo'

search = ARGV.join(' ')
file = File.expand_path('~/Documents/Library.xlsx')
libx = Roo::Spreadsheet.open(file)
lib = libx.sheet(libx.default_sheet).to_csv
lib = lib.split("\n")
lib.each do |line|
  puts line if line.include? search
end
